client/my-sites/plugins/main.jsx

```diff
@@ -386,6 +387,7 @@ const PluginsMain = React.createClass( {
 		return (
 			<Main className={ containerClass }>
 				{ this.renderDocumentHead() }
+				<QuerySites allSites />
```

Comment:
This is my fault. I added it and we merged it to production like half year ago, because I wasn't aware the redux version of the sites list wasn't supposed to be used yet. Since it was already in production, and it's not using any of the missing bits, and those missings bits were supposed to be added pretty soon, we decided to let it there and keep using it. Everything we need in this component is already built in our redux state tree & associated reducers, so it's working fine ... but if you think it would be better to rollback this to use the old sites-list, you probably know better than I do